keps/sig-scheduling/4815-dra-partitionable-devices/README.md

```diff
@@ -252,222 +252,470 @@ non-goals of this KEP.
 
 ## Proposal
 
-The basic idea is the following:
+On a high level, this proposal is based around two new concepts:
+
+1. Capacity pool, which is a construct for defining the capacities available for
+   sharing among partitions of a single physical device. Drivers/controllers will
+   typically create a capacity pool that specifies all the available capacities of
+   a physical device and then define the partitions available for DRA. Each
+   partition will specify how much capacity it will consume from the capacity
+   pool, which allows the scheduler to keep track of available capacity in
+   a pool and decide whether a partition can be allocated. A device will
+   separatly set the capacities that is published for the partition.
```

Comment:
s/separatly/separately/

s/set the capacities that is published/define the capacities that are accessible to the user/